apis/keda/v1alpha1/triggerauthentication_types.go

```diff
@@ -110,7 +110,8 @@ const (
 // AuthPodIdentity allows users to select the platform native identity
 // mechanism
 type AuthPodIdentity struct {
-	Provider PodIdentityProvider `json:"provider"`
+	Provider   PodIdentityProvider `json:"provider"`
+	IdentityID string              `json:"identityId"`
```

Comment:
This change makes mandatory this field, and it'll break other pod identities

Reply:
```suggestion
	// +optional
	IdentityID string              `json:"identityId"`
```

Reply:
```suggestion
	// +optional
	IdentityID string `json:"identityId"`
```

Reply:
Ok good catch. I will reapply this directly locally on my own branch and make a single commit for the entire PR remarks